offer-manager-domain/src/test/groovy/life/qbic/business/offers/OfferViewDtoSpec.groovy

```diff
@@ -0,0 +1,20 @@
+package life.qbic.business.offers
+
+
+import spock.lang.Specification
+
+/**
+ * <p>Checks for equality and hashcode to work</p>
+ *
+ * @since 1.1.0
+ */
+class OfferViewDtoSpec extends Specification {
+
+    def "equality is correct for equal offer vies"() {
+        //TODO
+    }
+
+    def "different objects are not equal"() {
+        //TODO
+    }
+}
```

Comment:
todo?

Reply:
yes I wanted to tackle that after the general idea was improved. This is a lot of work because you have to create customers and all the other things to compare two offers and or product items